### ISSUE TITLE
[Bug]: mouse hover generates too much traffic

### ISSUE THREAD

**Issue by allevo** [reactjs/react.dev#6711]
### Summary

When the mouse hovers on something, a download is performed. The cache is not used at all (max-age=0 with must-revalidate). This increase the network usage a lot.

### Page

https://react.dev/

### Details

https://github.com/reactjs/react.dev/assets/1054125/0d3868ac-10b0-499a-8368-baa71baaaacb

**Comment by BartoszKlonowski** [comment]
Hello @allevo!
I can no longer reproduce this issue (Mozilla's Firefox). Can you check on your side or specify which browser you see this on (if)?

**Comment by allevo** [comment]
I can reproduce it with Chrome, Safari, and Arc browser.

**Comment by BartoszKlonowski** [comment]
Exactly, this will be reproducible with any browser, as this seems to be default behavior of Next.js and the prefetching behavior that can't be simply disabled for hover: https://nextjs.org/docs/pages/api-reference/components/link#prefetch
